src/internal/minikubetestenv/deploy.go

```diff
@@ -32,8 +32,28 @@ const (
 	licenseKeySecretName   = "enterprise-license-key-secret"
 )
 
-// defensively lock around helm calls
-var mu sync.Mutex
+var (
+	mu sync.Mutex // defensively lock around helm calls
+
+	computedPachAddress *grpcutil.PachdAddress
```

Comment:
since we depend on the address to assign node ports, we compute it once to avoid potential race conditions